.github/workflows/lint.yml

```diff
@@ -0,0 +1,41 @@
+name: Lint
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+jobs:
+  rubocop:
+    name: Rubocop (runner)
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v1
+      - name: rubocop
+        uses: reviewdog/action-rubocop@v1
+        with:
+          rubocop_version: 0.89.1
+          rubocop_extensions: rubocop-performance:1.7.1 rubocop-rails:2.5.2
+          github_token: ${{ secrets.github_token }}
+          reporter: github-pr-check
+  brakeman:
+    name: Brakeman
+    runs-on: ubuntu-18.04
+    # runs only in main repo, this check doesn't work in forks
+    if: github.event.pull_request.head.repo.full_name == github.repository
```

Comment:
Only devmasx/brakeman-linter-action doesn't work on forks, right? I don't think we **have** to use it. Just the build failure on `brakeman` failing should be enough. Using this means brakeman will be skipped on all PRs.

Reply:
I'll check this. I thought it doesn't work just other way around (if you open PR in custom forks).

Reply:
Any way I can fall back to simple custom action for now. Those pre-used actions do report problems inline as checks annotations. That's the only benefit.